cfme/infrastructure/datastore.py

```diff
@@ -448,3 +462,12 @@ class Utilization(CFMENavigateStep):
 
     def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.monitoring.item_select("Utilization")
+
+
+@navigator.register(Datastore, 'ManagedVMs')
```

Comment:
**Optional**

Since your navigator class name is same as destination's registered name, you don't need to define it, you could just do - 
```
@navigator.register(Datastore):
class ManagedVMs(CFMENavigateStep):
    ...
```